CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## Version 1.1.0 - Update release - 2023/04
+
+- Add support for Python versions:
```

Comment:
As we did not make any code changes for these versions, perhaps it should just say "Add supported Python versions:" or "Declare support for Python versions:"?